src/main/java/com/simpligility/maven/plugins/android/phase09package/ApkMojo.java

```diff
@@ -21,6 +21,7 @@
 import com.android.sdklib.build.DuplicateFileException;
 import com.android.sdklib.build.SealedApkException;
 import com.simpligility.maven.plugins.android.AbstractAndroidMojo;
+import com.google.common.io.Files;
```

Comment:
hm... sorry.. looks like its not a fork but a powerful replacement. I guess thats fine.